enterprise/frontend/src/embedding-sdk/types/index.ts

```diff
@@ -18,14 +18,14 @@ type BaseSDKConfig = {
   allowConsoleLog?: boolean;
 };
 
-export type SDKConfigWithJWT = BaseSDKConfig & {
-  jwtProviderUri: string;
+export type SDKConfigWithAuthProviderUri = BaseSDKConfig & {
```

Comment:
```suggestion
export type SDKConfigWithAuthProvider = BaseSDKConfig & {
```
Nit: I'd keep that out of the name as it was before